src/qweb/compiler.ts

```diff
@@ -143,11 +152,13 @@ export class QWebCompiler {
   target = new CodeTarget("main");
   templateName: string;
   template: string;
+  translateFn?: (s: string) => string;
   ast: AST;
   staticCalls: { id: string; template: string }[] = [];
 
-  constructor(template: string, name?: string) {
+  constructor(template: string, name?: string, translateFn?: (s: string) => string) {
     this.template = template;
+    this.translateFn = translateFn;
```

Comment:
would probably be better to have a fallback: `this.translateFn = translateFn || word => word;` so you don't need to check later on if it exists